curveball/templates/curveball/curveballblock.html

```diff
@@ -1,19 +1,24 @@
 {% load markup %}
-{{block.body|markdown}}
 
-{% comment %}
-{% extends "base.html" %}
-{% load bootstrap %}
-{% load accessible %}
-
-{% block content %}
-
-  {% if block.curveballsubmission_set.count > 0 %}
-    {% for curveball in block.curveballsubmission_set.all %}
-       {{curveball.title}}
-       {{curveball.explanation}}
-    {% endfor %}
-  {% endif %}
-
-{% endblock %}
-{% endcomment %}
\ No newline at end of file
+  {% if block.curveball_one %}
```

Comment:
Given the possible use case of always having 3 options, this logic will always publish the first curveball. Are you adding in a template tag or some other functionality to render only the selected curveball for a given user?